main.html

```diff
@@ -1,70 +1,63 @@
 <!DOCTYPE html>
-<html>
-    <meta name="viewport" content="width = device-width, initial-scale = 1">
-    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
-    <link rel="stylesheet" href="public/css/app.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
-    <script src="https://rawgithub.com/hilios/jQuery.countdown/master/dist/jquery.countdown.min.js"></script>
-    <script src="public/js/app.js"></script>
-<head>
-    <title>Pomodoro Clock</title>
-</head>
-<body>
-    <div class="container">
-        <div class="row">
-            <div class="col-xs-12 text-center">
-                <div class="title">
-                    <h1>Pomodoro Clock</h1>
-                </div>
-            </div>
-        </div>
-        <div class="col-md-4 col-md-offset-1 col-xs-12 text-center ctl">
-            <div class="col-md-12 col-xs-6 box">
-                <div>
-                    <h2>Break Length</h2>
+<html ng-app="app">
+    <head>
+        <meta name="viewport" content="width = device-width, initial-scale = 1">
+        <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
+        <link rel="stylesheet" href="public/css/app.css">
+        <script src = "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.9/angular.min.js"></script>
+        <script src="public/js/app.js"></script>
+        <title>Pomodoro Clock</title>
+    </head>
+    <body>
+        <div class="container" ng-controller='ctrl'>
+            <h1 class="text-center">Pomodoro Clock</h1>
+            <div class="col-xs-12 text-center box">
+                <div class="col-xs-6 box--ctrl">
+                    <h2>{{ labels.break }} Length</h2>
+                    <div>
+                        <span>
+                            <button type="button" class="btn"
```

Comment:
Put your ng-click logic in a function, and call that function when user clicks.  It has 2 benefits:
1)  Your logic could grow later on, would be a hassle to change it in the html, especially if u have same logic in multiple spots.
2)  Code reuse.  If its a function, you can use it anywhere within the controller's scope, so you can replace both lines with similar logic with this function.